### ISSUE TITLE
Feature request: command add-network-policy

### ISSUE THREAD

**Issue by dustpuppyNGTI** [nulldriver/cf-cli-resource#37]
I use CF for hosting an app that I would like to push using a zero downtime deploy. App deploys depend on the manifest for configuration and as far as I can tell an add-network-policy feature inside the manifest does not exist.

I thought it might be possible to use this plugin to configure the policy directly after the deploy, not perfect but better than nothing. So I would like this feature to be added:

    - put: cf-add-network-policy
      resource: cf-env
      params:
        command: add-network-policy
        source_app: my-app
        destination_app: some-other-app
        protocol: tcp
        port: 1234

Link to relevant docs: https://docs.cloudfoundry.org/devguide/deploy-apps/cf-networking.html

**Comment by TheKoguryo** [comment]
In some case, destination app is located in different org, space from source app. For example, source app is web app and so is located in isolation segment. Destination app is was app and so is located in the other space.

I think it is required to set a destionation_space_name, destination_org_name additionally.

```
- put: cf-add-network-policy
  resource: cf-env
  params:
    command: add-network-policy
    source_app: my-app
    destination_app: some-other-app
    destination_space_name: space of some-other-app
    destination_org_name: org of some-other-app
    protocol: tcp
    port: 1234
```

**Comment by patrickcrocker** [comment]
At the time when I added the `add-network-policy` command to cf-cli-resource the version of the cf cli was [v6.37.0](https://github.com/cloudfoundry/cli/releases/tag/v6.37.0) and did not include the ability to target a destination space or org.

The `-s` and `-o` flags were introduced in cf cli [v6.42.0](https://github.com/cloudfoundry/cli/releases/tag/v6.42.0).  I'll add these as parameters in the next release!

**Comment by patrickcrocker** [comment]
Implemented in [v2.17.0](https://github.com/nulldriver/cf-cli-resource/releases/tag/v2.17.0)!
